Microsoft/WebTemplateStudio
# post gen modal as design

Based on the user testing, the current post generation modal is confusing to use. Thus, Christina came up with a new design. This PR is to make post generation functions as the new design. 

**How to test:**
1. try to generate a project with or without Azure Services 
2. when click **create project**, the post generation modal should look as expected


**Expected ( with Azure Services selected):**
![post-modal](https://user-images.githubusercontent.com/12874317/60932118-6cb02c80-a271-11e9-9570-82b60c2fbfc7.PNG)
